.gitignore

```diff
@@ -0,0 +1,3 @@
+
+.DS_Store
+.idea/
```

Comment:
These are OK but instead of putting these in every project, I think mac users should put `.DS_Store` in their `~/.gitignore` and solve it in one place. Same thing for `.idea` for intellij users.

Reply:
never knew this, will consider it